rest/model/build.go

```diff
@@ -121,3 +122,33 @@ type APITaskCache struct {
 	Activated       bool                    `json:"activated"`
 	FailedTestNames []string                `json:"failed_test_names,omitempty"`
 }
+
+type APIVariantTasks struct {
+	Variant      *string
+	Tasks        []string
+	DisplayTasks []APIDisplayTask
+}
+
+func APIVariantTasksBuildFromService(v patch.VariantTasks) APIVariantTasks {
```

Comment:
Just curious, do we intend to migrate existing  BuildFromService/ToService to use this new format?

Reply:
Most likely not. Changing existing routes will change their public interface in small ways (usually relating to null handling/omitempty) which will break consumers